.github/workflows/checks.yml

```diff
@@ -72,14 +71,32 @@ jobs:
       contents: read
       checks: write
 
+  # The actions runner controller does not support using multiple labels to target the runners so we have to create separate jobs for self-hosted runners.
+  # https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/using-actions-runner-controller-runners-in-a-workflow#using-runner-scale-set-names
+  e2e-quick-start-custom-runner:
+    strategy:
+      matrix:
+        provider:
+          - Nutanix
+          # Uncomment below once we have the ability to run e2e tests on other providers via self-hosted runners.
+          # - Vsphere (example)
+      fail-fast: false
+    uses: ./.github/workflows/e2e-custom-runner.yml
+    with:
+      provider: ${{ matrix.provider }}
+      focus: Quick start
+    secrets: inherit
+    permissions:
+      contents: read
+      checks: write
+
   e2e-self-hosted:
     strategy:
       matrix:
         provider:
           - Docker
           # Uncomment below once we have the ability to run e2e tests on other providers from GHA.
           # - AWS
-          # - Nutanix
       fail-fast: false
     uses: ./.github/workflows/e2e.yml
     with:
```

Comment:
I wonder if we can use a new input to the e2e test to change the `runs-on` and use a conditional to check the provider to set the runner label in the inputs here?

Reply:
Pushed #755 that does this.